src/content/en/shows/designer-vs-developer/podcast/DVDS2E04-learn-how-to-design.md

```diff
@@ -0,0 +1,50 @@
+project_path: /web/_project.yaml
+book_path: /web/shows/_book.yaml
+description: Yasmine talks learning to design.
+
+{# wf_updated_on: 2017-09-11 #}
+{# wf_published_on: 2017-09-11 #}
+{# wf_podcast_audio: https://storage.googleapis.com/designer-vs-developer-podcast/DVDS2E04-learn-how-to-design.mp3 #}
+{# wf_podcast_duration: 00:11:02 #}
+{# wf_podcast_fileSize: 15930283 #}
+{# wf_podcast_subtitle: Yasmine talks learning to design. #}
+{# wf_featured_image: /web/shows/designer-vs-developer/podcast/images/dvd-s02-e04-cover-large.jpg #}
+
+# Learn how to design {: .page-title }
+
+Welcome to Designer Vs. Developer, a show that tries to solve the 
+challenges faced in the industry by opening a conversation between 
+the two, providing take aways, solutions to workflows, tools and 
+discussions on everyday struggles. 
+
+Mustafa speaks to Yasmine Evjen, a Developer Programs Engineer on 
+the Google Developer Relations Team, about learning design skills and 
+debate if technical expertise help or hinder the creative process.
```

Comment:
Change to:
debate whether technical expertise helps or hinders the create process.